vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.h

```diff
@@ -50,6 +50,10 @@ struct CompositionBaseComponentView : public IComponentView,
   void updateBorderProps(
       const facebook::react::ViewProps &oldViewProps,
       const facebook::react::ViewProps &newViewProps) noexcept;
+  void updateStyleProps(
```

Comment:
Maybe this should be called updateShadowProps, since that what it seems to handle. -- There are a lot of style props.

Reply:
I agree, best to keep them in groupings so we have view components can be more granular about opting-in.